share/spice/libraries/triggers.txt

```diff
@@ -0,0 +1,11 @@
+go
+wordpress
+atom
+homebrew
```

Comment:
Let's remove this unless we are planning to deprecate [Homebrew ](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Homebrew.pm)